low-level/files/opt/seagate/sspl/bin/deploy_sspl

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+set -e -u -o pipefail
+
+PRODUCT="LDR_R2"
+RPMS_PATH=
+SSPL_BASE_DIR="/opt/seagate/cortx/sspl"
+
+
+usage()
+{
+    echo "\
+    Standalone SSPL deploy script.
+    (Bounded to single node deployment)
+    Usage:
+         $0
+            [-P|--product <LDR_R2>]
+            [-R|--local_rpms_path <local RPMS path>]
+            [-h|--help]
+    OPTION:
+    -P      <PRODUCT>           Product name (LDR_R1 | LDR_R2)
+    -R      <LOCAL RPMS PATH>   Path to local RPMS. If this is not passed,
+                                SSPL RPMS will be installed using yum repos.
+    "
+    exit 0
+}
+
+parse_args()
+{
+    while [[ $# -gt 0 ]]; do
+        case "$1" in
+        -h|--help)
+            usage ;;
+        -P|--product)
+            [ -z "$2" ] && echo "ERROR: Product not provided" && exit 1;
+            PRODUCT="$2"
+            shift ;;
+        -R|--local_rpms_path)
+            [ -z "$2" ] && echo "ERROR: Local RPMS path not provided" && exit 1;
+            RPMS_PATH="$2"
+            shift 2 ;;
+        *)
+            echo "ERROR: Unknown option provided: $1"
+            exit 1 ;;
+        esac
+    done
+}
+
+parse_args "$@"
+
+[ "$PRODUCT" != "LDR_R2" ] && {
+    echo "ERROR: deploy_sspl doesn't have support for product '$PRODUCT'"
+    exit 1
+}
+
+# If not local RPMS path is not passed, this will install
+# sspl specific RPMS using yum repos.
+if [ -n "$RPMS_PATH" ]
+then
+    echo -e "INFO: Installing SSPL RPMS from local path... \ncd ${RPMS_PATH}"
+    cd $RPMS_PATH
+    sudo yum install -y cortx-libsspl_sec-[12]* \
+                        cortx-libsspl_sec-method_none* \
+                        cortx-sspl-[12]* cortx-sspl-test*
```

Comment:
can we maintain specific rpm names, excluding version ofcourse, or currently it would end up installing cortx-sspl-cli also & alike additionally i guess.

Reply:
cortx-sspl-cli won't be installed since here version is specified to the package name.
Now I introduced slight changes to pick only v2.0.0 package.